workspaces/liferay-ticket-workspace/readme.markdown

```diff
@@ -0,0 +1,24 @@
+# Liferay Ticket Workspace
+
+## Deployment
+
+First we have to enable the below feature flag:
+
+    # Enabling attachments for object entries
+    feature.flag.LPS-174455 = true
+
+Then install the below client extensions in the exact order below :
```

Comment:
"Then install the below client extensions in the exact order below:"
No space between `below` and `:`